docs/streams/developer-guide.html

```diff
@@ -943,10 +943,14 @@
                 <pre class="brush: java;">
                        KStream&lt;byte[], String&gt; stream = ...;
                        stream.print();
-
-                       // Several variants of `print` exist to e.g. override the default serdes for record keys
-                       // and record values, set a prefix label for the output string, etc
-                       stream.print(Serdes.ByteArray(), Serdes.String());
+                    
```

Comment:
For doc changes: I think we also need to add a paragraph in upgrade section talking about the removal of `serdes` in print, and that users should not try to use `serdes` anymore but to override the mapper function if their key / values default `toString` function are not good enough (think about `Bytes`).